src/Tests/Microsoft.NET.Sdk.Razor.Tests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -174,7 +174,7 @@ public void Build_DoesNotGenerateManifestWhen_NoStaticResourcesAvailable()
             var projectDirectory = CreateAspNetSdkTestAsset(testAsset);
 
             var build = new BuildCommand(projectDirectory);
-            build.Execute().Should().Pass();
+            build.Execute("/p:ScopedCssEnabled=false").Should().Pass();
```

Comment:
This is just because I added scoped css files and they will be included as SWA, so I'm disabling it here to keep the test the same.